data/en/mbs_0202.json

```diff
@@ -0,0 +1,478 @@
+                                        "value": 0,
+                                        "id": "turnover-answer",
+                                        "condition": "equals"
+                                    }],
+                                    "block": "confirm-zero-turnover-block"
+                                }
+                            },
+                            {
+                                "goto": {
+                                    "block": "exports-block"
+                                }
+                            }
+                        ]
+                    },
+                    {
+                        "type": "ConfirmationQuestion",
+                        "title": "Turnover",
+                        "id": "confirm-zero-turnover-block",
+                        "skip_conditions": [{
+                            "when": [{
+                                "id": "turnover-answer",
+                                "condition": "greater than",
+                                "value": 0
+                            }]
+                        }],
+                        "questions": [{
+                            "type": "General",
+                            "answers": [{
+                                "type": "Radio",
+                                "id": "confirm-zero-turnover-answer",
+                                "q_code": "d40",
+                                "options": [{
+                                        "label": "Yes this is correct",
+                                        "value": "Yes"
+                                    },
+                                    {
+                                        "label": "No I need to change this",
+                                        "value": "No"
+                                    }
+                                ],
+                                "mandatory": true
+                            }],
+                            "id": "confirm-zero-turnover-question",
+                            "title": "For the period {{ format_conditional_date (answers['period-from'], exercise.start_date)}} to {{ format_conditional_date (answers['period-to'], exercise.end_date)}}, the value of the total turnover was <em>£0</em>, is this correct?"
+                        }],
+                        "routing_rules": [{
+                                "goto": {
+                                    "when": [{
+                                        "value": "No",
+                                        "id": "confirm-zero-turnover-answer",
+                                        "condition": "equals"
+                                    }],
+                                    "block": "turnover-block"
+                                }
+                            },
+                            {
+                                "goto": {
+                                    "block": "exports-block"
```

Comment:
changes

Reply:
@shahi645 ?

Reply:
should be "goto" : { "block" : "chnges or significant changes"

Reply:
"goto": {"block": "changes-in-turnover-block"}